src/hotspot/share/gc/shenandoah/shenandoahFreeSet.hpp

```diff
@@ -81,6 +79,12 @@ class ShenandoahFreeSet : public CHeapObj<mtGC> {
 public:
   ShenandoahFreeSet(ShenandoahHeap* heap, size_t max_regions);
 
+  // How many regions dedicated to GC allocations (for evacuation or promotion) are currently free?
```

Comment:
I agree with these comments - let's delete the question marks?

Reply:
Fixed.